tests/08-native-runs/14-smallprng/Makefile

```diff
@@ -0,0 +1,9 @@
+CONTIKI_PROJECT = test-smallprng
+all: $(CONTIKI_PROJECT)
+
+TARGET = native
+
+MODULES += os/services/unit-test
+
+CONTIKI = ../../..
```

Comment:
You don't need to define `CONTIKI` nowadays, the build system will figure that out itself. (So `include ../../../Makefile.include` saves a line and avoids a definition.)

Reply:
Neat!